lib/src/components/modal/ModalHeaderTabs.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { CloseButton } from '../../modules/closeButton/CloseButton';
+
+export function ModalHeaderTabs({ children, headerText, onCloseClick }) {
+  return (
+    <div className="modal-header__tabs">
+      <div className="modal-header__tabs-text">{headerText}</div>
+      {children}
```

Comment:
💚 I wonder if it's easier app side to include the `NavigationItem` code here rather than passing it in.

My concern is that doing it app side multiple times will result in variations in how it looks, and will take longer.

That said, it may not be sane to implement!

Reply:
Personally I like what `FileCard` does and set a prop called something like `navigationItem`. Take a look at that example and maybe it'll help.